src/ILCompiler.WebAssembly/src/ILCompiler.WebAssembly.csproj

```diff
@@ -53,10 +60,13 @@
     <Compile Include="CodeGen\ILToWebAssemblyImporter.cs" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="..\..\..\packages\llvmsharp\3.9.1-rc3\lib\netstandard1.1\LLVMSharp.dll">
+    <Content Include="..\..\..\packages\llvmsharp\5.0.0\lib\netstandard1.1\LLVMSharp.dll">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="..\..\..\packages\libllvm\3.9.1\content\libLLVM-x64.dll">
+    <Content Include="..\..\..\packages\libllvm4\4.0.1.3\runtimes\osx\native\libLLVM.dylib" Condition="'$(HostOS)' == 'OSX'">
```

Comment:
Nit:  The directory name should be `osx-x64` ... but since this temporary, you can leave this as is.